single_level.py

```diff
@@ -50,21 +50,21 @@ def makeRow(resource):
 	global row
 	row = []
 	publish = get_single_value(resource, "publish")
-	title = get_single_value(resource, "title")
-	resourceId = get_single_value(resource, "id_0")
+	title = get_single_value(resource, "title").encode('utf-8')
+	resource_ids = get_single_value(resource, "id_0")
```

Comment:
What's the logic in changing this to plural? All the other values here are singular, so it seems like this should be too?